### PR TITLE
Qualify `gazebo::util` in `using namespace` directives.

### PR DESCRIPTION
**Pull request by suertreus**
This avoids ambiguity in the presence of a top-level namespace named `util`, e.g. https://godbolt.org/z/WMfseEGPf.

### CODE REVIEW

**Review by ahcorde — APPROVED** [review]

**Comment by suertreus** [comment]
Is there a way to retry to CI on this?  I don't think this namespace qualification change is actually responsible for test failures or segfaults.

**Comment by ahcorde** [comment]
I was waiting for other maintainer to give you another approval. @scpeters or @WilliamLewww do you mind to take a look ?

**Review by scpeters — APPROVED** [review]

**Pull request merged**
